### ISSUE TITLE
Sidebar images violates WCAG 1.1.1

### ISSUE THREAD

**Issue by auchenberg** [Microsoft/vscode-docs#2615]
image-alt: Images must have alternate text (7) WCAG 1.1.1

Url: https://code.visualstudio.com/docs/editor/versioncontrol

Links:
- https://dequeuniversity.com/rules/axe/3.2/image-alt?application=msftAI
- https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html

Code paths
```

Path | #connect-twitter-link > a[target="_blank"] > img
-- | --
Snippet | <img src="/assets/community/sidebar/twitter.svg">
Path | a[href$="feed\.xml"][target="_blank"] > img
Snippet | <img src="/assets/community/sidebar/rss.svg">
Path | #connect-stack-overflow > a[target="_blank"] > img
Snippet | <img src="/assets/community/sidebar/stackoverflow.svg">
Path | #connect-twitter > a[target="_blank"] > img
Snippet | <img src="/assets/community/sidebar/twitter.svg">
Path | #connect-uservoice > a[target="_blank"] > img
Snippet | <img src="/assets/community/sidebar/github.svg">
Path | #connect-github > a[target="_blank"] > img
Snippet | <img src="/assets/community/sidebar/issue.svg">
Path | #connect-youtube > a[target="_blank"] > img
Snippet | <img src="/assets/community/sidebar/youtube.svg">
```

**Comment by gregvanl** [comment]
@auchenberg I think you'll need to change your comment since the issue 839 is no longer in the vscode-website repo.
